Gitlab 0.1.6 (#64)
port-labs/Port-Ocean: integrations/gitlab/changelog/1.improvement.md

> Upgraded ocean for using the optimized `on_resync` generator

Reviewer: ```suggestion
Upgraded ocean version to use the optimized `on_resync` generator
```
